Moodlet/app/src/test/java/com/cmput3owo1/moodlet/models/MoodEventAssociationTest.java

```diff
@@ -0,0 +1,33 @@
+package com.cmput3owo1.moodlet.models;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MoodEventAssociationTest {
+
+    private MoodEvent moodEvent = new MoodEvent(EmotionalState.HAPPY);
+    private String username = "UwU";
+
+    private MoodEventAssociation mockMoodEventAssociation() {
+        return new MoodEventAssociation(moodEvent, username);
+    }
+
+    @Test
+    public void testConstructor() {
+        MoodEventAssociation moodEventAssociation = new MoodEventAssociation(new MoodEvent(EmotionalState.CONFUSED), "Test");
+        assertNotNull(moodEventAssociation);
```

Comment:
Might wanna assert that the object actually has CONFUSED and "test" inside of it
Something like
moodEventAssociation.getUsername() == "Test

Reply:
Added constructor content validation tests